src/config.template.yaml

```diff
@@ -26,6 +26,10 @@ sound:
   media_output_device:
   media_default_volume:
 
+  # Some devices clip off the beginning/end of audio files being played back. The
+  # value is the number of seconds of silence to pad the beginning and end of the audio
+  playback_padding: 0
```

Comment:
Good idea! Although the cause should be fixed. Now it feels like linux/systemd when we're supposed to fix someone else's mess. Well, since there's probably no easy way to fix the damn CHIP audio, I'm glad we have this!